src/components/AdditionalInfo/AdditionalInfo.jsx

```diff
@@ -48,8 +65,9 @@ export default class AdditionalInfo extends React.Component {
     });
 
     const trigger = (
-      <button
-        type="button"
+      <a
```

Comment:
> conditionally use that component as an `<a>` tag with `role="button"`

I don't know what the condition _should_ be but it seems missing.

Reply:
I had spoken with @noahgelman prior and he said that his interpretation was that the conditional was based on whether there was expandable content or not. However, as I started working through that I couldn't think of a situation where we wouldn't have expandable content, unless it was future planning to have an alternate implementation of these. After discussing with our team engineering lead we couldn't figure out what the intended conditional was either, so I was going to defer to you all for what the intended outcome was here. Whatever is decided, I'm happy to implement.